XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -255,7 +255,8 @@ the following attributes: Height, MaxHeight, Length, MaxLength, Diameter, and Ma
                                         Click="Animated_GotItem" 
                                         GotFocus="Animated_GotItem" 
                                         HorizontalAlignment="Stretch"
-                                        Loaded="GetButtonSize"/>
+                                        Loaded="GetButtonSize"
+                                        Foreground="{StaticResource SystemBaseHighColor}"/>
```

Comment:
Removing the foreground did not seem to make a difference, was there a specific reason why this was added? 😅

Reply:
+1 - this seems unrelated to the issue. Maybe this is leftover from a different fix? In either case, please remove.